here/traffic/sql/function_detach_partition.sql

```diff
@@ -0,0 +1,35 @@
+-- Detach partitioned tables in here.ta 
+-- input yyyy text
+CREATE OR REPLACE FUNCTION here.detach_partition_ta(
```

Comment:
Should this file be dropped from the PR if it's not planned for regular use?

Reply:
hmmm I think we can keep it in PR but drop from the database, then we will always have the code to recreate similar procedures. It can probably go in a separate folder or sth tho